siman/project_funcs.py

```diff
@@ -4716,7 +4716,7 @@ def run_OMC( cl_defect, cl_ideal, defect_atoms = None, defect_occ = None, ise =
 
 
 def run_OMC_sol(cl_defect, cl_ideal, defect_atoms = None, defect_occ = None, ise = None, suf = '', up = 0, gmt = 0, soluted_atom = None, e_s = None, 
-    add_loop_dic = None, cluster = 'cee-omc'):
+    add_loop_dic = {}, cluster = 'cee-omc'):
```

Comment:
this is bad, never use {} or [] as default arguments, this will lead to errors!
add in function:

if add_loop_dic is None:
 add_loop_dic = {}

Reply:
done